kuwala-io/kuwala
# 🧱 Data blocks and ❄️ Snowflake connector

In this PR we added the functionality to add data blocks based on the connected data sources to the canvas.

Our community member, [Eteims](https://github.com/EteimZ), also added a new data warehouse connector, Snowflake! 🎉

The currently supported data warehouses are: Postgres, Snowflake, and BigQuery

To test this PR, do the following steps:

1. Launch Docker
2. To start the backend, follow the instructions under `kuwala/core/backend/README.md` (make sure to pull the latest version of the backend image)
3. To start the frontend, follow the instructions under `kuwala/core/canvas/README.md`

Closes #83 
Closes #114
